ruby/test/fixtures/test/flaky_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+class FlakyTest < Minitest::Test
+  def test_dummy
+    assert true
+  end
+
+  def test_flaky
+    assert_equal '1', ENV['FLAKY_TEST_PASS']
+  end
+end
```

Comment:
Is this dead code?  Since the added test uses `test/failing_test.rb`.  Looks like you might have tried to use this to `Simulate another worker successfuly retrying all errors` but forgot to remove this code when you couldn't get that working.

Reply:
Yes, totally dead code. I'll 🔥